sys/shell/cmds/mtd.c

```diff
@@ -0,0 +1,497 @@
+/*
+ * Copyright (C) 2025 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
```

Comment:
```suggestion
/*
 * SPDX-FileCopyrightText: 2025 ML!PA Consulting GmbH
 * SPDX-License-Identifier: LGPL-2.1-only
 */
```

Even though this PR was started before the whole SPDX migration, perhaps you could update the license header according to #21515 nevertheless.